src/lib/make-request.ts

```diff
@@ -1,11 +1,10 @@
-import { loadBundle } from "edgespec"
 import type { Middleware } from "edgespec/middleware"
 
+// @ts-expect-error (otherwise this will throw during `edgespec codegen`)
+import bundle from "../../dist/bundled-routes.js"
```

Comment:
Is there an upstream issue we can link to in edgespec?

Reply:
this isn't really an edgespec issue, `npx edgespec codegen route-types` uses the TypeScript AST/checker so your code has to type check successfully before you can generate types

Reply:
we could change the ordering so that instead `npx edgespec bundle` runs before `npx edgespec codegen`, but then `npm run codegen` will fail if you run it outside of `npm run build`.

Reply:
For `build:*` scripts I see them as a unit, e.g., `npm run build` will work but individual steps may not work if you run them manually out of order.

Reply:
personally I would rather keep the ability to run `npm run codegen` outside of build; I commonly use it to update route types since the route types are used in tests (typed-axios)
a single @ts-expect-error seems like a minor tradeoff for that